app/frontend/src/features/profile/ProfileForm.tsx

```diff
@@ -67,7 +68,7 @@ export default function EditProfileForm() {
       lat: user?.lat ?? undefined,
       lng: user?.lng ?? undefined,
       radius: user?.radius ?? undefined,
-      // avatarUrl: user?.avatarUrl ?? undefined,
+      avatarKey: undefined,
```

Comment:
Does this need to be `avatarKey: user.avatarKey` - otherwise maybe the avatar will be unset if the user is changing things that aren't the avatar?

Reply:
But that might mess with the thumbnail...

Reply:
Maybe the solution would be an `initialImage` prop or something

Reply:
That's a good point.
It seems that the backend needs the avatarkey to update the profile image. I assumed that if it was undefined or empty then the backend would just ignore it.

@aapeliv